### PR TITLE
Show complete file path of active scene in window title bar

### PR DESCRIPTION
**Pull request by thomas-goerlich**
Hello,
i added a small suggestion by @wyattbiker in issue #22911 to add the complete file path to the window title. Not sure if this is the right / best way to do it so please let me know if i can make it better :)
regards
Thomas

godot_pathInToolbar

### CODE REVIEW

**Comment by Calinou** [comment]
I'm not entirely sure about this, as this path can potentially be quite long, which makes it difficult to read. Maybe we should display the path relative to the project root instead?

**Comment by thomas-goerlich** [comment]
I could do that, sure. I was just going for what was asked for in the issue. Should i change it or are there more opinions needed?

**Comment by KoBeWi** [comment]
> Should i change it or are there more opinions needed?

You could make it as separate commit (not PR) and later it should either be squashed or reverted.

**Comment by akien-mga** [comment]
I also think the full path is overkill, we already have the project name which should be a good indication to the user of which project (and thus project folder on their filesystem) they are editing.

Having the scene path relative to `res://` might be useful though, especially if you have different scenes with the same name in different folders.

**Comment by thomas-goerlich** [comment]
I updated the code to only include the project path:
godot-title

**Review by Calinou — APPROVED** [review]

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -206,7 +206,7 @@ void EditorNode::_update_title() {
 	String title = appname.empty() ? String(VERSION_FULL_NAME) : String(VERSION_NAME + String(" - ") + appname);
 	String edited = editor_data.get_edited_scene_root() ? editor_data.get_edited_scene_root()->get_filename() : String();
 	if (!edited.empty())
-		title += " - " + String(edited.get_file());
+		title += " - " + edited.get_base_dir().replace_first("res://", "").plus_file(edited.get_file());
```
Comment (akien-mga) [inline editor/editor_node.cpp]: `edited.replace_first("res://")` should be enough, no?

**Comment by groud** [comment]
I also don't think the full scene path is necessary too. It's not that useful in the top bar.
That might be added as a tooltip on the scenes tabs, though it should be made sure that look good along with the scene preview.

File: editor/editor_node.cpp (hunk 1 above)
Comment (thomas-goerlich) [reply editor/editor_node.cpp]: Oh, yes sorry! You are absolutely right. Fixed it :)

**Comment by akien-mga** [comment]
We discussed this briefly in a PR meeting, and overall the proposed change doesn't seem very popular. A better alternative would be to add the scene path to the tooltip when you hover the scene tabs.

But in any case, this is a typical case of something that should go through https://github.com/godotengine/godot-proposals to see what interest there is in the community for the proposed changed. Otherwise it's really hard for us to decide whether to merge a PR without any metrics on whether users want it.

**Pull request closed without merging**